pkg/minikube/cluster/cluster_linux.go

```diff
@@ -21,7 +21,10 @@ import (
 	"path/filepath"
 
 	kvm "github.com/dhiltgen/docker-machine-kvm"
+	local "github.com/dims/docker-machine-local"
```

Comment:
@r2d4 : Ack, will drop my repo and copy the code directly into minikube. Since we don't have https://github.com/docker/machine/blob/master/drivers/generic/generic.go in our vendor'ed stuff. The difference is minimal and once we have that vendored, we can get rid of the local.go.